dev/integration_tests/keyboard_hot_restart/lib/main.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+// If true, the app autofocuses a text field, making the software keyboard visible.
+// See:
+// //dev/devicelab/lib/tasks/keyboard_hot_restart_test.dart
+const bool forceKeyboard = true;
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(home: MyHomePage());
+  }
+}
+
+class MyHomePage extends StatelessWidget {
+  const MyHomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // Print the app's view insets.
+    // The test uses the bottom value to check whether the keyboard is visible.
+    // ignore: avoid_print
+    print('viewInsets: ${MediaQuery.of(context).viewInsets}');
```

Comment:
can we do something like: 

```
if viewInsets == .zero {
  Text("Keyboard is dismissed")
} else {
  Text("Keyboard is presented")
}
```

Then in the test, we check the Text on screen?

Reply:
That's a great idea, updated!